anyway/localization.py

```diff
@@ -222,3 +222,179 @@ def get_city_name(symbol_id, lang: str = "he") -> Optional[str]:
+        text += f"{join_hebrew_strings([severity_fatal_count_text, severity_severe_count_text, severity_light_count_text])}"
+        text = f".{text}"
+        return text
+
+    def accident_count_by_severity_hebrew():
+        date_range = output.get('meta').get('dates_comment').get('date_range')
+        accident_count_by_severity = list(filter(lambda widget: widget.get('name') == 'accident_count_by_severity', output.get('widgets')))[0]
+        severity_light_count = accident_count_by_severity.get('data').get('items').get('severity_light_count')
+        if severity_light_count == 0:
+            severity_light_count_text = ''
+        elif severity_light_count == 1:
+            severity_light_count_text = 'קלה אחת'
+        else:
+            severity_light_count_text = f'{severity_light_count} קלות'
+
+        severity_severe_count = accident_count_by_severity.get('data').get('items').get('severity_severe_count')
+        if severity_severe_count == 0:
+            severity_severe_count_text = ''
+        elif severity_severe_count == 1:
+            severity_severe_count_text = 'קשה אחת'
+        else:
+            severity_severe_count_text = f'{severity_severe_count} קשות'
+
+        severity_fatal_count = accident_count_by_severity.get('data').get('items').get('severity_fatal_count')
+        if severity_fatal_count == 0:
+            severity_fatal_count_text = ''
+        elif severity_fatal_count == 1:
+            severity_fatal_count_text = 'קטלנית אחת'
+        else:
+            severity_fatal_count_text = f'{severity_fatal_count} קטלניות'
+        total_accidents_count = accident_count_by_severity.get('data').get('items').get('total_accidents_count')
+
+
+        if output.get('meta').get('resolution') == 'STREET':
+            text = f"בעיר {output.get('meta').get('location_info').get('yishuv_name')} ברחוב {output.get('meta').get('location_info').get('street1_hebrew')} "
+        elif output.get('meta').get('resolution') == 'SUBURBAN_ROAD':
+            text = f"בכביש {output.get('meta').get('location_info').get('road1')} במקטע {output.get('meta').get('location_info').get('road_segment_name')} "
+        else:
+            raise Exception(f"cannot convert to hebrew for resolution : {output.get('meta').get('resolution')}")
+        text += f"בשנים {date_range[0]}-{date_range[1]} התרחשו {total_accidents_count} תאונות דרכים, מתוכן: "
+        text += f"{join_hebrew_strings([severity_fatal_count_text, severity_severe_count_text, severity_light_count_text])}"
+        text = f".{text}"
+        return text
+
+
+    result = {}
+    try:
+        result['injured_count_by_severity'] = injured_count_by_severity_hebrew()
+        result['accident_count_by_severity'] = accident_count_by_severity_hebrew()
+        result['most_severe_accidents_table'] = most_severe_accidents_table_hebrew()
+        result['head_on_collisions_comparison_percentage_hebrew'] = head_on_collisions_comparison_percentage_hebrew()
+
```

Comment:
what use try catch? what if any of the widgets is not preset? I saw you handled it within the functions but is it best practice?